rules-presets/modify-traffic/modify-reponse-body.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Modify Response Body
+description: "Modify an HTTP response by inserting HTML elements while using Fiddler's rules."
+slug: adv_modify_response_body
+position: 10
+---
+
+# Modifying Response Bodies
+
+One of the most common testing practices is quickly changing a piece of code or a resource and rebuilding it to visualize the results. This can be time-consuming if the rebuild also involves deploying your application online.
+
+With Fiddler Everywhere, you can quickly modify the content of the received HTTP responses by inserting, substituting, or entirely changing the targeted entries.
+
+## Creating an "Insert HTML" Rule
+
+Create an "Insert HTML" rule by setting the following actions through the [Rules Builder]({%slug modify-traffic-get-started%}).
+
+1. Create a matching condition that uses the "When **all these conditions** are met **any number of times**" pattern. 
+
+1. Match by a **URL** that uses a string value to match the desired URL. For example: **example.com**.
+
+1. Create an **Update Response Body** action and use the **Find and Replace** string modifier.
+
+This sample Fiddler rule finds a selected H1 HTML element and inserts a logo before it.
+
+
+![Creating "Insert HTML" rule](../../images/advanced/adv-modify-insert-html.png)
+
+Once the rule is created, enable the **Rules** tab, toggle the rule switch, and start capturing traffic.
+
+![Activating the "Insert HTML" rule](../../images/advanced/adv-modify-insert-html-active.png)
+
+Download a ready-to-use <a href="https://github.com/telerik/fiddler-everywhere/tree/master/rules/modify-insert-html" target="_blank">"Insert HTML"</a> rule as a FARX file, which you can import through the Rules toolbar.
+ 
+ 
+ ## Creating an "Insert HTML (ReGex)" Rule
```

Comment:
Note the extra space in front. It breaks your rendering.
![image](https://github.com/telerik/fiddler-everywhere-docs/assets/3539221/4dec8253-6d94-4aba-8b15-18e122b92ebd)

```suggestion
## Creating an "Insert HTML (RegEx)" Rule
```